Bio/GenBank/Record.py

```diff
@@ -165,33 +165,33 @@ class Record(object):
     SEQUENCE_FORMAT = "%" + str(GB_SEQUENCE_INDENT) + "s"
 
     def __init__(self):
-        self.locus = ''
-        self.size = ''
-        self.residue_type = ''
+        self.accession = []
+        self.base_counts = ''
+        self.comment = ''
+        self.contig = ''
         self.data_file_division = ''
         self.date = ''
+        self.db_source = ''
+        self.dblinks = []
         self.definition = ''
-        self.accession = []
+        self.features = []
+        self.gi = ''
+        self.keywords = []
+        self.locus = ''
         self.nid = ''
+        self.organism = ''
+        self.origin = ''
         self.pid = ''
-        self.version = ''
+        self.primary = []
         self.projects = []
-        self.dblinks = []
-        self.db_source = ''
-        self.gi = ''
-        self.keywords = []
+        self.references = []
+        self.residue_type = ''
         self.segment = ''
+        self.sequence = ''
+        self.size = ''
         self.source = ''
-        self.organism = ''
         self.taxonomy = []
-        self.references = []
-        self.comment = ''
-        self.features = []
-        self.base_counts = ''
-        self.origin = ''
-        self.sequence = ''
-        self.contig = ''
-        self.primary = []
+        self.version = ''
```

Comment:
Did you actually change anything here, or just re-sort the attributes alphabetically? That makes sense, but would have been clearer as a self-contained commit is it is unrelated to the comment parsing work. If you're comfortable with git and are happy to revise the branch history, that would be great - otherwise don't worry about it.